### PR TITLE
Sets TTL support for mongodb state store to enabled

### PR DESCRIPTION
**Pull request by JoshVanL**
Implemented [here](https://github.com/dapr/components-contrib/pull/2535).

### CODE REVIEW

**Comment by berndverst** [comment]
@hhunter-ms code PR is merged. @JoshVanL to make sure this docs PR matches the final implementation.

**Comment by JoshVanL** [comment]
As I understand, this is all the docs I need. Unless I need to add some releases notes, or similar, somewhere?

**Review by hhunter-ms — APPROVED** [review]

**Comment by hhunter-ms** [comment]
> As I understand, this is all the docs I need. Unless I need to add some releases notes, or similar, somewhere?

@JoshVanL should the [MongoDB component spec](https://docs.dapr.io/reference/components-reference/supported-state-stores/setup-mongodb/) be updated to reflect this?

**Comment by JoshVanL** [comment]
@hhunter-ms thanks, I agree. We should update that page with a mention of TTL like the others. Leave that to me

**Comment by JoshVanL** [comment]
Done

File: daprdocs/content/en/reference/components-reference/supported-state-stores/setup-mongodb.md [hunk 1]
```diff
@@ -112,6 +112,10 @@ The username is `admin` by default.
 
 {{< /tabs >}}
 
+### TTLs and cleanups
+
+This state store supports [Time-To-Live (TTL)]({{< ref state-store-ttl.md >}}) for records stored with Dapr. When storing data using Dapr, you can set the `ttlInSeconds` metadata property to indicate when the data should be considered "expired".
```
Comment (hhunter-ms) [inline daprdocs/content/en/reference/components-reference/supported-state-stores/setup-mongodb.md:117]: Can you also add `ttlInSeconds` to the example and the spec metadata table above?

**Review by hhunter-ms — CHANGES_REQUESTED** [review]

File: daprdocs/content/en/reference/components-reference/supported-state-stores/setup-mongodb.md (hunk 1 above)
Comment (JoshVanL) [reply daprdocs/content/en/reference/components-reference/supported-state-stores/setup-mongodb.md:117]: Mongodb doesn't currently support setting the ttl on the component spec so I left those out.

**Review by hhunter-ms — APPROVED** [review]

**Pull request merged**